### PR TITLE
Update index.html.erb

### PR DESCRIPTION
**Pull request by allalala**
Added administrative and policy language.
@vzvenyach please let us know if this looks OK.

### CODE REVIEW

File: app/views/logins/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+
+<p>Once you’re ready, continue with GitHub authorization (only public account information will be requested). If you encounter any issues, let us know at <%= mail_to 'micropurchase@gsa.gov' %>.</p>
 
-<p>Once you’re ready, continue with GitHub authorization (only public account information will be requested).</p>
 
 <a class='usa-button' href="/auth/github">Authorize with GitHub</a>
+<p></p>
+<p></p>
+<p>Administrative Notes</p>
+<p>None of the content in this GitHub repository is considered proprietary or procurement-sensitive information.
```
Comment (vzvenyach) [inline app/views/logins/index.html.erb]: @allalala I think we need to specify which GitHub repository (https://github.com/micropurchase).

Additionally, I wonder whether we should focus on having two warnings: (1) at signup; and (2) at the time of bidding.

At signup, it seems that we'd say that we will only use this information to authenticate users and to communicate on an as-needed basis.

At time of bidding, I'd think we'd want to provide a warning describing how the bid information (and the bidder information) would be used for that bid/opportunity.

Thoughts>

Comment (allalala) [inline app/views/logins/index.html.erb]: Sounds good. Will make the changes to the copy on the bidding page.

**Comment by vzvenyach** [comment]
@adelevie any idea why there's this security warning here?

**Comment by adelevie** [comment]
I _think_ the fork @allalala PR'd from is older/out-of-sync.

**Comment by allalala** [comment]
yes, @adelevie I think this is a problem with my fork. Can you show me how to fix this on Monday?

**Comment by adelevie** [comment]
Yep!

**Comment by adelevie** [comment]
closing in favor of https://github.com/18F/micropurchase/pull/117 (merged)

**Pull request closed without merging**